src/utilities/templates/flatpages/default.html

```diff
@@ -1,11 +1,8 @@
-{% extends "base_sidebar.html" %}
```

Comment:
This will need a few adjustment once you fix you base.html

Reply:
Made a few adjustments to default. 
- Removed unnecessary tags
- removed js code (i think it just adds html tags to text), because summernoteis now being used